modules/ROOT/pages/backup-restore/copy-database.adoc

```diff
@@ -212,11 +212,14 @@ For more information on the block format, see xref:database-internals/store-form
 
 [NOTE]
 ====
-In addition to the `--from-pagecache=<size>` option, a new option `--max-off-heap-memory` is introduced in Neo4j 2025.01.
+In Neo4j 2025.01, the `--from-pagecache=<size>` option is replaced with a dual-named option: `--from-pagecache, --max-off-heap-memory=<size>`.
+This change helps you to control off-heap memory rather than just allocate cache.
 
-You can still use the `--from-pagecache` option to speed up the copy operation by specifying how much cache to allocate when reading the source.
-The `--from-pagecache` should be assigned whatever memory you can spare since Neo4j does random reads from the source.
-Note that the `--from-pagecache` option is considered to be deprecated.
+Both options control the same underlying parameter, which determines the maximum amount of off-heap memory available for reading the source database during the copy process.
```

Comment:
It's not only about reading the source database, it's the total amount of off-heap memory the process will use, both for reading and for writing.